src/administration/integrations/github.md

```diff
@@ -50,26 +50,27 @@ Optional parameters:
 * `fetch-branches`: Track and deploy branches (true by default)
 * `build-pull-requests`: Track and deploy pull-requests (true by default)
 
-Note that if your repository belongs to an organisation, use ``--repository=ORGANISATION/REPOSITORY``.
+> **Note:**
+> If your repository belongs to an organization, use ``--repository=ORGANISATION/REPOSITORY``.
 
 ### 3. Add the webhook
 
 Copy the Payload URL that is returned by the previous command.
 
 Go to your GitHub repository and click `Settings`. Select the *Webhooks
 and Services* tab and click `Add webhook`. Paste the Payload URL, Choose
-"Send me everything" for the events you want to receive. and click
+"Send me everything" for the events you want to receive and click
 `Add webhook`.
 
 You can now start pushing code, creating new branches or opening pull
 requests directly on your GitHub repository.
 
-> **note**
-> If you have created your account using the github oAuth Login in order to use the Platform CLI you will need to setup a
-> password which you can do by visiting this page [https://accounts.platform.sh/user/password](https://accounts.platform.sh/user/password)
+> **Note:**
+> If you have created your account using the GitHub oAuth Login, in order to use the Platform CLI, you will need to setup a
+> password here: [https://accounts.platform.sh/user/password](https://accounts.platform.sh/user/password).
```

Comment:
As long as we're modifying it, though, the "here" type link is generally considered bad practice.  Let's fix that.  Instead, let's make "setup a password" the link text.